onnxruntime/core/graph/contrib_ops/bert_defs.cc

```diff
@@ -302,6 +302,81 @@ ONNX_MS_OPERATOR_SET_SCHEMA(
           AttentionTypeAndShapeInference(ctx, past_input_index);
         }));
 
+ONNX_MS_OPERATOR_SET_SCHEMA(
+    DecoderMaskedMultiheadAttention, 1,
+    OpSchema()
+        .SetDoc(Attention_ver1_doc)
```

Comment:
>Attention_ver1_doc [](http://example.com/codeflow?start=16&length=18)

update the doc for the 1 token attention

Reply:
Done